communications.md

```diff
@@ -2,95 +2,62 @@
-
-Opt-out: Leave projects that you are no longer interested in or obligated to.
-
-## Second line of Communication: Meeting
+Our Asana workspace is commonly used for project management e.g task allocation, task state (in progress or completed), and any other process we deem it fit to be used for (e.g Office Availability, calender scheduling, etc.)
+Opt-in to join projects/tasks that you are interested in or obligated to.
 
-If a meeting is necessary, budget the amount of time needed appropriately. Each meeting is the collective productivity value of all its participants. Consider each participant to be on average contributing $200/hr; it is always good to judge the need/value of each meeting against the economic value of the meeting (what other productive tasks could these participants be doing?).
 
-#### - Meetings & Events Tool: Google Calendar -
+#### Meetings
+If a meeting is necessary, we have a meeting space that can be pre-booked and used appropriately. Please be mindful of the duration time as we presently have limited private space for meetings and others might be waiting in line.
 
-Opt-in: Subscribe to calendars you are interested in or obligated to.
+###### Events Tool: Google Calendar -
+Opt-in to subscribe to calendars you are interested in or obligated to.
+You can set up meetings by using event invitations and book one of the available rooms in the google invitation to avoid disappointment.
 
-Set up meetings: Use event invitations to set up a meeting and book one of the available rooms in the google invitation to avoid disappointment.
+###### Preparing for & During Meetings: 
+To ensure that meetings are productive and are respectful of everyone's time and productivity, feel free to try out the steps below:
 
-Preparing for & During Meetings: To ensure that meetings are productive and are respectful of everyone's time and productivity, you may do the following:
-
-1. Aggressively timebox them. If a meeting was set to be an hour-long, ask if it can be shortened to 30 minutes; if it was set to be 30 minutes long, ask if it can be shortened to 15 minutes... better yet, see if you can do it without a meeting.
+1. Timebox meeting to ensure the duration is convenient for every or most participants.
 2. Write down the agenda, and prepare every participant by sending the agenda out before the meeting.
-3. Instead of abstractly talk about a problem in a hollow boardroom, demonstrate the problem, perhaps on-site or screen-shared. Do not stretch beyond the agreed timebox, make a decision and move on.
-4. Ensure there are clear and concise takeaways. Every meeting must have a clear outcome.
-5. One of the participants should concisely summarize the meeting and its key points.
-
-Opt-out: Unsubscribe from calendars you are no longer interested in.
-
-#### - Meeting Documentation: Slack -
+3. Demonstrate the problem, perhaps on-site or screen-shared. Try to keep to the proposed timeboxed duration for the meeting.
```

Comment:
first half 👍 , makes it very concise. second half is redundant since we already have timebox point from 1.

Reply:
We can take it out as it could be redundant. Just wanted it to be some sort of reminder on the duration time when demonstrating a problem.